src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.ps1

```diff
@@ -2994,6 +2995,66 @@ function Test-VirtualMachineGetStatus
     }
 }
 
+<#
+.SYNOPSIS
+Test Virtual Machines
```

Comment:
Improve this description. It does not describe what the test does.

Reply:
Improved. As a note, my description is the only one in the virtualmachinetests file. This means that we need to decide as a team a directive for descriptions since no one has been doing it before. Resolving.